README.md

```diff
@@ -31,16 +31,14 @@ which can be used to add custom widgets.
 
 * [Source Install](#markdown-header-source-install)
 
-    * [Prerequisites](#markdown-header-prerequisites)
+[Testing](#testing)
+
+[Folder Structure](#folder-structure)
 
     * [Building from Source](#markdown-header-building-from-source)
 
 [Usage](#markdown-header-usage)
 
-[Documentation](#markdown-header-documentation)
-
-[Testing](#markdown-header-testing)
-
 [Folder Structure](#markdown-header-folder-structure)
```

Comment:
folder structure is repeated

Reply:
Resolved f1610c19f83028910c2a08c48ab26044e71ac3ae